src/model/reference.ml

```diff
@@ -90,21 +92,22 @@ let match_extra_odoc_reference_kind (_location as loc) s :
 
    A secondary reason to delay parsing, and store strings in the token list, is
    that we need the strings for user-friendly error reporting. *)
-let match_reference_kind location s : Paths.Reference.tag_any =
+let match_reference_kind location s : reference_kind =
   match s with
-  | None -> `TUnknown
-  | Some s -> (
+  | `None -> `TUnknown
+  | `Prefixed s | `Old_prefix s -> (
```

Comment:
Some places have `` `Prefixed `` and not `` `Old_prefix `` but it's otherwise not necessary. Old prefixes could just be passed in as new prefixes.
It's just self-documentation added to the old code.